helm-charts/interoperator/templates/operator_api_app.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    service: {{ .Release.Name }}-op-apis-service
+  name: {{ .Release.Name }}-op-apis-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: {{ .Values.operator_apis.service.type }}
+  ports:
+  - name: http
+    port: 80
+    targetPort: op-apis-port
+  - name: https
+    port: 443
+    targetPort: op-apis-port
+  selector:
+    app: {{ .Release.Name }}-op-apis-app
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  labels:
+    service: {{ .Release.Name }}-op-apis-service
+    {{- if .Values.cluster.ssl }}
+    garden.sapcloud.io/purpose: managed-cert
+    {{- end }}
+  name: {{ .Release.Name }}-op-apis-service-ingress
+  namespace: {{ .Release.Namespace }}
+spec:
+  rules:
+  - host: {{ .Values.cluster.op_apis_host }}
```

Comment:
since this is a mandatory value to be passed, can we also construct it somehow if not provided based on the broker host!